Library/Backing+PledgeAmount.swift

```diff
@@ -9,4 +9,13 @@ extension Backing {
       subtractingShippingAmount: self.shippingAmount.flatMap(Double.init)
     )
   }
+
+  /// Returns the bonus support amount subtracting the reward minimum from the total pledge amount
+  public var bonusSupportAmount: Double {
+    guard let reward = self.reward else { return 0 }
+
+    let bonusSupportAmount = Decimal(self.pledgeAmount) - Decimal(reward.minimum)
+
+    return (bonusSupportAmount as NSDecimalNumber).doubleValue
```

Comment:
Don't know if there is historical context to this, but why not return:
`self.pledgeAmount - reward.minimum `

Reply:
We just wrap these in `Decimal` when doing addition/subtraction for accuracy given the trouble with floating-point arithmetic. However, in searching for some supporting docs to confirm this, it seems we might not be initializing `Decimal` quite right here. I'll update this!

Reply:
Re slack convo - will update in separate PR.